tests/src/whisk/core/controller/test/MetaApiTests.scala

```diff
@@ -353,7 +365,7 @@ class MetaApiTests extends ControllerTestCommon with WhiskMetaApi with BeforeAnd
                                 "pkg" -> s"$systemId/proxy".toJson,
                                 "action" -> "export_c".toJson,
                                 "content" -> metaPayload(
-                                    m.method.name.toLowerCase,
+                                    m.method.value,
```

Comment:
Why change this? `m.method.name.toLowercase` seems more speaking to me than to rely on `value` returning a lower cased name?